### PR TITLE
Bitwarden

### PR DESCRIPTION
**Pull request by pernielsentikaer**
## Description

Added items folder names if any

## Screencast

CleanShot 2022-09-21 at 07 08 42@2x

### CODE REVIEW

**Comment by pernielsentikaer** [comment]
The next step could be to use a dropdown to filter items + maybe add the folder to the keyword attribute.

**Comment by andreaselia** [comment]
Looks good to me, nice work @pernielsentikaer 👍

File: extensions/bitwarden/src/search.tsx [hunk 1]
```diff
@@ -54,16 +54,18 @@ export default function Search() {
...
     items: [],
+    folders: [],
     isLocked: false,
     isLoading: true,
   });
 
   async function loadItems(sessionToken: string) {
     try {
+      const folders = await bitwardenApi.listFolders(sessionToken);
       const items = await bitwardenApi.listItems(sessionToken);
```
Comment (pomdtr) [inline extensions/bitwarden/src/search.tsx:67]: We should make those two requests in parallel here.
The startup time of the bitwarden extension is already pretty slow, so the last thing we want is too increase it.

**Review by pomdtr — CHANGES_REQUESTED** [review]

**Comment by pomdtr** [comment]
Thanks for the contribution @pernielsentikaer !

**Comment by pomdtr** [comment]
Looks good to me ! Btw we should use the new usePromise api to simplify the code :)

**Comment by pomdtr** [comment]
Also what do you think of adding a dropdown to filter by folder ?

**Comment by pernielsentikaer** [comment]
Sounds like a good idea with usePromise, but I don't think I have enough time to do it at the moment, maybe a project for another?

[Dropdown](https://github.com/raycast/extensions/pull/2967#issuecomment-1253216533) is next step and will be in another PR when I have time, I don't think I can do it now so another might take that too

**Pull request merged**